web-react-ts/src/pages/dashboards/Navigation.tsx

```diff
@@ -23,11 +23,10 @@ const Navigator = () => {
     variables: { email: user?.email },
     onCompleted: (data) => {
       const church = data.memberByEmail.stream_name
-
       setCurrentUser({
         ...currentUser,
         id: data.memberByEmail.id,
-        fellowship: data.memberByEmail?.fellowship.id,
+        fellowship: data.memberByEmail?.fellowship?.id,
```

Comment:
Why would there be a scenario where a member doesn't belong to a fellowship. That's what I want to understand from this

Reply:
thats a fair point, i think what i should rather do is assign myself to a fellowship

Reply:
Yes so do that and take this question mark out